DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: desc
 Title: Manipulate DESCRIPTION Files
-Version: 1.1.1.9003
+Version: 1.1.1.9004
```

Comment:
Don't update this please.

Reply:
I have updated with the current master to 1.2.0